### PR TITLE
Replace queries with hardcoded "monica" database name to use the current default connection database

### LINKED ISSUE #1213: SQL export assumes a "monica" database
https://github.com/monicahq/monica/blob/a6ac8a8f5fa526f4f903d80a9c3896b670c6f0bd/app/Jobs/ExportAccountAsSQL.php#L86

Seems it would be more appropriate to look for the current connection database (or the environment `DB_DATABASE`).

### PR DESCRIPTION
**Pull request by tomzx**
Fixes #1213

### CODE REVIEW

**Comment by djaiss** [comment]
Nice work @tomzx !!

File: app/Helpers/DBHelper.php [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+use Illuminate\Support\Facades\DB;
+
+class DBHelper
+{
+    public static function getTables()
+    {
+        return DB::table('information_schema.tables')
+            ->select('table_name')
+            ->where('table_schema', '=', static::getDefaultConnectionConfiguration('database'))
```
Comment (asbiin) [inline app/Helpers/DBHelper.php]: You can do: `DB::connection()->getDatabaseName()` to get the database name.

Comment (tomzx) [reply app/Helpers/DBHelper.php]: :+1:

**Comment by asbiin** [comment]
@tomzx have you seen my comment about `array_filter()` function ?

**Comment by tomzx** [comment]
I saw it, but I forgot apply a fix for it. I'll do that ASAP.

**Comment by tomzx** [comment]
@asbiin After reviewing the code, I suggest we remove the filtering altogether...

https://github.com/monicahq/monica/blob/0468845208280bc49952be5bf43cd9bc17146e55/app/Jobs/ExportAccountAsSQL.php#L135-L139

So it's doing a bunch of logic just to basically get the "accounts" string into `$tableName`...

**Comment by asbiin** [comment]
Thanks @tomzx !

**Pull request merged**